server/src/Patterns.ts

```diff
@@ -18,9 +18,36 @@ const patterns: string[] = [
 	"<head(?:.|)+?>(?:(?:\\s|\\S)+?(?=<\/head>))<\/head>",
 	"<html(?:.)+?>"
 ];
-
 export const pattern: RegExp = new RegExp(patterns.join('|'), 'ig');
 
+const nonDescriptiveAlts = [
+	"alt=\"image\"",
+	"alt=\"picture\"",
+	"alt=\"logo\"",
+	"alt=\"icon\"",
+	"alt=\"graphic\"",
+	"alt=\"an image\"",
+	"alt=\"a picture\"",
+	"alt=\"a logo\"",
+	"alt=\"an icon\"",
+	"alt=\"a graphic\"",
+];
+const nonDescriptiveAltsTogether = new RegExp(nonDescriptiveAlts.join("|"), "i");
+
+const badAltStarts = [
+	"alt=\"image of",
+	"alt=\"picture of",
+	"alt=\"logo of",
+	"alt=\"icon of",
+	"alt=\"graphic of",
+	"alt=\"an image of",
+	"alt=\"a picture of",
+	"alt=\"a logo of",
+	"alt=\"an icon of",
+	"alt=\"a graphic of",
+];
```

Comment:
I was considering ways to make a loop to create this list with image, picture, logo, icon and graphic as the base strings, but the a/an problem stumped me.